include/quda_arpack_interface.h

```diff
@@ -7,10 +7,7 @@
 #include <color_spinor_field.h>
 #include <dirac_quda.h>
 #include <vector>
-
-//#ifdef PRIMME_LIB
-//#include "primme.h"
-//#endif
+#include <algorithm>
```

Comment:
Can you move these `std` headers to where they are needed rather than blanket inclusion in this header?